spec/support/page_helpers.rb

```diff
@@ -41,4 +45,5 @@ def reload_page_until_timeout!(text:, as_link: false, table: nil, with_reindex:
   # rubocop:enable Metrics/CyclomaticComplexity
   # rubocop:enable Metrics/MethodLength
   # rubocop:enable Metrics/PerceivedComplexity
+  # rubocop:enable Metrics/ParameterLists
 end
```

Comment:
allow us to use this method with a specific element on the page (useful for verifying specific workflow completion)